api/awards.py

```diff
@@ -0,0 +1,86 @@
+from flask import g, make_response, jsonify, request
+from flask.views import MethodView
+from sqlalchemy.sql.expression import false
+import time, datetime
+
+
+from service.award_service import add_award_return_id, add_employee_award_return_id
```

Comment:
This service is not in the PR/commit. Did you merge this PR without tests passing?